### ISSUE TITLE
funky sorting on loan list

### ISSUE THREAD

**Issue by cjconroy** [ArctosDB/arctos#1361]
If you review loan items in Arctos, the list is different if you have it on 100 or 250 per page. I was double checking the list of specimens loaned, and looking at it with 100 per page, it looked like lots of specimens were missing. I had forgotten there were greater than 100 specimens. I sorted the list more than once. Somehow it does not sort the entire list by cat number when on 100 per page, just those 100 that happen to make it onto the page under some other criteria. See attached images. One is for when set for 100 per page and sorted by MVZ number, the other for the 250 per page, again sorted by MVZ number. Can this be fixed?

<img width="328" alt="screen shot 2017-02-02 at 10 18 58 am 1" src="https://user-images.githubusercontent.com/5749672/33914502-51d466f2-df53-11e7-8238-c8b78d85117c.png">
<img width="328" alt="screen shot 2017-02-02 at 10 18 58 am" src="https://user-images.githubusercontent.com/5749672/33914504-51e84186-df53-11e7-9620-8ae2aced3d9c.png">

**Comment by dustymc** [comment]
Seems to be working properly for me and your screenshots are identical?? I need more information.

**Comment by cjconroy** [comment]
Pull up loan 2016.8363.Mamm. Go to review items. Select show 100 per page. Sort on cat num. I get the numbers from top 150022, 150023, 150024, 153746, 160713, etc. 
Click on show 500 per page. Sort again on cat num. I now get the numbers from top 150021, 150022, 150023, 150024, 153745, 153746, 160713, etc. Note the addition of 150022 and 153745. New attachments. 

<img width="334" alt="500 per page" src="https://user-images.githubusercontent.com/5749672/34008775-4f556034-e0bb-11e7-86eb-6c079091fcfc.png">
<img width="353" alt="100 per page" src="https://user-images.githubusercontent.com/5749672/34008776-4f7a6fb4-e0bb-11e7-9245-25c560fa1bb5.png">

**Comment by cjconroy** [comment]
Perhaps related in some weird way, I re-found the loan by navigating to one of the specimens. On that detail page, I scrolled down to the links for loans for that specimen. It pulls up a new window with the loan, but oddly it shows (1 item), when the loan actually has 195. I pulled up the loan through transactions. See screen shots. 
<img width="564" alt="screen shot 2017-12-14 at 10 49 37 am" src="https://user-images.githubusercontent.com/5749672/34009151-a1c80596-e0bc-11e7-8b92-acf9506446d9.png">
<img width="523" alt="screen shot 2017-12-14 at 10 43 35 am" src="https://user-images.githubusercontent.com/5749672/34009152-a1df8b30-e0bc-11e7-9b71-85786cef338a.png">
